### PR TITLE
[DXP Cloud] LRDOCS-9468 Backup inconsistencies limitation concerns

### PR DESCRIPTION
**Pull request by Alec-Shay**
See: https://issues.liferay.com/browse/LRDOCS-9468

As previously explained by Support, the primary concerns about the wording of this limitation were particularly that:
- It may give the impression that DXP Cloud is especially limited in this sense (when other solutions may be similarly limited)
- It should describe what users can do to mitigate this risk

Hopefully this wording sufficiently addresses these concerns while the limitation still exists.

### CODE REVIEW

File: docs/dxp-cloud/latest/en/platform-services/backup-service/backup-service-overview.md [hunk 1]
```diff
@@ -116,6 +116,10 @@ See [Configuration via LCP.json](../../reference/configuration-via-lcp-json.md)
 
 Determining how frequently backups are created and removed can help protect your data and optimize storage.
 
+```warning::
+   Scheduled backups created while your Liferay instance is running and data is actively changing risk creating inconsistent data. Configure your backup schedule to create backups during times with reduced activity to mitigate the risk of data inconsistency. To ensure a completely consistent backup, coordinate with your database administrator to freeze updates while you perform a `manual backup <./backup-service-overview.md#creating-a-manual-backup>`__.
+```
```
Comment (jrhoun) [inline docs/dxp-cloud/latest/en/platform-services/backup-service/backup-service-overview.md:121]: @Alec-Shay is the issue here:

a. If you **schedule** a backup while your liferay instance is running "bad things could happen".
or
b. If a backup you previously scheduled executes while your liferay instance is running, "bad things could happen".

?

Comment (Alec-Shay) [reply docs/dxp-cloud/latest/en/platform-services/backup-service/backup-service-overview.md:121]: Well, in both cases, yes

But the particular issue this warning is speaking to is more in reference to case B. I think we wouldn't use the term "schedule" for the more active scenario.

Comment (Alec-Shay) [reply docs/dxp-cloud/latest/en/platform-services/backup-service/backup-service-overview.md:121]: To clarify (if it helps), the proposed strategy to mitigate this is: 
- In the long term (to **reduce** but not entirely **prevent** possible inconsistencies), schedule your backups for a time of reduced activity if possible (still possible something happens at the wrong time)
- In the short term, if you need 100% consistency, then do a manual backup, and actually freeze all changes to data for the duration (guaranteeing nothing happens at the wrong time)

**Comment by jrhoun** [comment]
Merged with wordsmithing here: https://github.com/liferay/liferay-learn/commit/666cdf8486c00a6ab6327d948157befa28c3214f

**Pull request closed without merging**